views/default/elements/typography.css.php

```diff
@@ -24,6 +24,25 @@
 	text-decoration: underline;
 }
 
+.elgg-anchor * {
+    display: inline;
+    vertical-align: middle;
+}
+
+.elgg-anchor * + .elgg-anchor-label {
+	margin-left: 5px;
+}
+
+.elgg-anchor-indicator:after {
+	content: attr(data-indicator);
```

Comment:
I'd rather somehow allow HTML in these, and name them badges: http://getbootstrap.com/components/#badges